src/scanner/main.rs

```diff
@@ -0,0 +1,476 @@
+extern crate shell_words;
+
+mod tui;
+
+mod input;
+use input::{Config, Opts, PortRange, ScanOrder};
+
+mod scanner;
+use scanner::Scanner;
+
+mod port_strategy;
+use port_strategy::PortStrategy;
+
+use colorful::Color;
+use colorful::Colorful;
+use futures::executor::block_on;
+use rlimit::Resource;
```

Comment:
This is kind of annoying that this entire file is flagged as "new", no idea how to fix it :(